src/chrome/content/rules/BergensTidende.xml

```diff
@@ -55,25 +115,43 @@
 
 -->
 <ruleset name="BT.no (partial)">
-  <target host="*.bt.no" />
-  <target host="bt.no" />
-		<!--
-			Redirects to http:
-						-->
-		<!--exclusion pattern="^http://kundesenter\.bt\.no/favicon\.ico" /-->
-		<!--exclusion pattern="^http://www\.bt\.no/$" /-->
-		<!--
-			Exceptions:
-					-->
-		<exclusion pattern="^http://kundesenter\.bt\.no/+(?!keypublisher_gui/|sitefiles/)" />
-		<exclusion pattern="^http://(?:www\.)?bt\.no/+(?!external/|skins/)" />
-		<!--
-			(Subject to broken MCB):
-						-->
-		<!--exclusion pattern="^http://(www\.)?bt\.no/+tips($|[?/])" /-->
-
-  <rule from="^http://(?:www\.)?bt\.no/" to="https://www.bt.no/" />
-
-	<rule from="^http://(kundesenter|statisk)\.bt\.no/"
-		to="https://$1.bt.no/" />
+
+	<target host="www.bt.no" />
+	<target host="adportal.bt.no" />
+	<target host="annonser.bt.no" />
+	<target host="beta.bt.no" />
+	<target host="blogg.bt.no" />
+	<target host="cis.bt.no" />
+	<target host="helios.bt.no" />
+	<target host="images.bt.no" />
+	<target host="kundeportal.bt.no" />
+	<target host="kundeweb.bt.no" />
+	<target host="kundewebtest.bt.no" />
+	<target host="lisa.bt.no" />
+	<target host="mediahash1.bt.no" />
+	<target host="mediahash2.bt.no" />
+	<target host="mediahash3.bt.no" />
+	<target host="mobil.bt.no" />
+	<target host="multimedia.bt.no" />
+	<target host="nyttig.bt.no" />
+	<target host="old.bt.no" />
+	<target host="reise.bt.no" />
+	<target host="statisk.bt.no" />
+
+	<rule from="^http://statisk\.bt\.no/" to="https://statisk.bt.no/" />
+
+	<rule from="^http://www\.bt\.no/(external/|public/|resources/|skins/)" to="https://www.bt.no/$1" />
```

Comment:
I will revisit all these norwegian news pages when i got time.
Might become a lot simpler if i extrapolate correctly from the aftenposten website.